_posts/2025-09-25-ethics-what-is-our-approach.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "Research Ethics: What should OLS take into account?"
+authors:
+- rivaquiroga
+- bethaniley
+- iramosp
+- yochannah
+image: PIC-URL-HERE
+photos:
+  name: AUTHOR-HERE
+  license: LICENCE-HERE
+  url: AUTHOR-ADD-URL-HERE
+---
+
+**TL;DR:** You're invited to share your thoughts about what is important in research ethics on [this GitHub discussion](https://github.com/orgs/open-life-science/discussions/1079), or via Slack in #general. 
+
+**Longer:** One of OLS's three core pillars of activity is research, focusing on creating and understanding a better evidence base for equity-serving open research. Some of our previous work includes Paz Bernaldo's [Open Seeds Impact paper](https://doi.org/10.5281/zenodo.15593896) (currently in press with Wellcome Open Research), and our Alan Turing Institute funded "Widening participation in data science" policy work. 
+
+## What we've done for ethics in the past
+
+For the work listed above, we used [Pearl IRB](https://www.pearlirb.com/)'s independent ethical review board. In both cases, we were given ethics "waivers". That is, we still had to fill out a large number of forms asking us what we were doing, what our consent form looked like, what our research protocols were, how we'd store our data, etc. - but after this was reviewed we weren't asked for much else, and were told we could go ahead with the work. The process worked well, and was reasonably fast and easy to use, prompting us to think of important things and making sure we were ready to really do a good job. 
+
+## We'd like to get a little bit more robust in our ethics processes. 
+OLS currently has research components built into two projects: [OSPARK]({% link ols-collab-research/index.md %}) and [Seeds to Systems]({% link seeds-to-systems-research/index.md %}). One thing we've learned while we were talking about this research is that our team comes from different backgrounds with very different experiences approaching when ethics reviews happen, and how they happen. Whilst many of us have conducted social research (e.g. interviews and focus groups), some of us have repeatedly been through ethics review processes, and others have done similar research in places where ethical review isn't expected outside a clinical setting. 
+
+## Where we stand
+I'm sure it won't surprise anyone when we say that research ethics is incredibly important to us as a team and as a community! In some research settings, the term "ethics" doubles as "ensuring legal liabilities are met". In others, it might mean the difference between publishing in a peer-reviewed journal (with proof of an ethics process) or not being allowed to publish (without ethics). To OLS, it's probably both of those, as well as being trustworthy, thoughtful researchers. That means we _want_ to be responsible for our own work and behaviour, regardless of any other review processes we might take part in. Which leads to the next question! 
```

Comment:
```suggestion
I'm sure it won't surprise anyone when we say that research ethics is incredibly important to us as a team and as a community! In some research settings, the term "ethics" doubles as "ensuring legal liabilities are met". In others, it might mean the difference between publishing in a peer-reviewed journal (with proof of an ethics process) or not being allowed to publish (without ethics). To OLS, it's probably both of those: but we also aim to be trustworthy, thoughtful researchers. That means we _want_ to be responsible for our own work and behaviour, regardless of any other review processes we might take part in. Which leads to the next question! 
```

Reply:
Rephrased this a little to move the emphasis on to being trustworthy and thoughtful (so it's clear this isn't an afterthought for OLS!)